src/core/Cutter.cpp

```diff
@@ -3465,6 +3475,17 @@ void CutterCore::addFlag(RVA offset, QString name, RVA size)
     emit flagsChanged();
 }
 
+/**
+ * @brief Gets all the flags present at a specific address
+ * @param addr The address to be checked
+ * @return String containing all the flags which are comma-separated
+ */
+QString CutterCore::listFlagsAsStringAt(RVA addr)
+{
+    RCore *core = Core()->core();
```

Comment:
Don't assign `CutterCore::core()` to `RCore *`.  It returns a RCoreLocked which functions like a scoped lock. Either use the correct type or in Cutter.cpp you can use `CORE_LOCK();`
```suggestion
    CORE_LOCK();
```

Reply:
Okay, I'll take a note of that.